packages/eslint-plugin/src/rules/prefer-optional-chain.ts

```diff
@@ -109,7 +116,105 @@ export default createRule<
     const seenLogicals = new Set<TSESTree.LogicalExpression>();
 
     return {
-      // specific handling for `(foo ?? {}).bar` / `(foo || {}).bar`
+      // specific handling for `if (foo) { foo.bar(); }` / `if (foo) foo.bar();`
+      'IfStatement[consequent.type=BlockStatement][consequent.body.length=1], IfStatement[consequent.type=ExpressionStatement]':
+        (
```

Comment:
[Style] Nit: no need to make these `: () =>` arrow function properties. They can be direct methods. That'd give it one less level of indentation.

```suggestion
      'IfStatement[consequent.type=BlockStatement][consequent.body.length=1], IfStatement[consequent.type=ExpressionStatement]'(
```

Reply:
Haha, not that I know of.